R/pkg/R/functions.R

```diff
@@ -3803,3 +3803,89 @@ setMethod("repeat_string",
             jc <- callJStatic("org.apache.spark.sql.functions", "repeat", x@jc, numToInt(n))
             column(jc)
           })
+
+#' is_grouping
+#'
+#' Indicates whether a specified column in a GROUP BY list is aggregated or not,
+#' returns 1 for aggregated or 0 for not aggregated in the result set.
+#'
+#' Same as \code{GROUPING} in SQL and \code{grouping} function in Scala.
+#'
+#' @param x Column to compute on
+#'
+#' @rdname is_grouping
+#' @name is_grouping
+#' @family agg_funcs
+#' @aliases is_grouping,Column-method
+#' @export
+#' @examples \dontrun{
+#' df <- createDataFrame(mtcars)
+#'
+#' # With cube
+#' agg(
+#'   cube(df, "cyl", "gear", "am"),
+#'   mean(df$mpg),
+#'   is_grouping(df$cyl), is_grouping(df$gear), is_grouping(df$am)
+#' )
+#'
+#' # With rollup
+#' agg(
+#'   rollup(df, "cyl", "gear", "am"),
+#'   mean(df$mpg),
+#'   is_grouping(df$cyl), is_grouping(df$gear), is_grouping(df$am)
+#' )
+#' }
+#' @note is_grouping since 2.3.0
+#' @seealso \link{cube}, \link{grouping_id}, \link{rollup}
```

Comment:
I'd simplify this - since this is the in `@family agg_funcs` generated doc already has links to most of these. unless there is a very direct relation, say to `grouping_id`, I don't think we need extra links here